fulp_modules/features/antagonists/infiltrator/infiltrators.dm

```diff
@@ -0,0 +1,74 @@
+/datum/antagonist/traitor/infiltrator
+	name = "Infiltrator"
+	antagpanel_category = "Infiltrator"
+	job_rank = ROLE_INFILTRATOR
+	antag_hud_name = "traitor"
+	hijack_speed = 1
+	show_name_in_check_antagonists = TRUE
+	show_to_ghosts = TRUE
+	antag_moodlet = /datum/mood_event/focused
+	preview_outfit = /datum/outfit/infiltrator
+	var/give_equipment = TRUE
+	var/list/infil_flavor
+
+/datum/job/infiltrator
+	title = ROLE_INFILTRATOR
+
+/datum/antagonist/traitor/infiltrator/proc/equip_infiltrator(mob/living/carbon/human/infiltrator = owner.current)
+	return infiltrator.equipOutfit(/datum/outfit/infiltrator)
+
+/datum/antagonist/traitor/infiltrator/on_gain()
+	if(give_objectives)
+		forge_traitor_objectives()
+	if(give_equipment)
+		equip_infiltrator(owner.current)
+	owner.current.mind.set_assigned_role(SSjob.GetJobType(/datum/job/infiltrator))
+	owner.current.mind.special_role = ROLE_INFILTRATOR
+	infil_flavor = strings("infiltrator_flavor.json", "infiltrator", "fulp_modules/strings/infils")
+	..()
+
+/datum/antagonist/traitor/infiltrator/greet()
+	. = ..()
+	owner.announce_objectives()
+
+/datum/antagonist/traitor/infiltrator/ui_static_data(mob/user)
```

Comment:
What's the difference between this and traitor that makes it worth giving its own ui_static_data?